.github/workflows/ci_action_noetic.yml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: CI-Noetic
```

Comment:
The diff seems to indicate this was was actually `git mv` from the original?

Reply:
I don't know. Maybe this file simply has more similarities with the original than `ci_action_melodic.yml`?